website_cookieconsent/static/src/js/cookieconsent/cookieconsent.js

```diff
@@ -0,0 +1,1143 @@
+/*!
+ * CookieConsent 3.0.0-rc.17
+ * https://github.com/orestbida/cookieconsent/tree/v3.0-beta
+ * Author Orest Bida
+ * Released under the MIT License
+ */
+var e, t;
+e = this, t = function(e) {
```

Comment:
This looks like it's still minified.

Reply:
Hey Tom, I only see minified version of the lib from the repo here => https://github.com/orestbida/cookieconsent/tree/master/dist

Reply:
Ah, you're right. Nevermind then.